nodewatcher/modules/defaults/network_profile/models.py

```diff
@@ -36,5 +36,6 @@ class RegistryMeta:
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('wifi-uplink', _("Use a wireless interface for uplink")))
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('no-wifi-ap', _("Disable wireless AP")))
 registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('no-wifi-mesh', _("Disable wireless mesh")))
+registration.point('node.config').register_choice('network.profile#profiles', registration.Choice('wifi-mesh-80211s', _("Use 802.11s for meshing")))
```

Comment:
@kostko, could we have here maybe `wifi-mesh-80211s` and `wifi-mesh-adhoc`? Is it possible to have profiles enabled by default? So that we would have `wifi-mesh-adhoc` by default enabled, for legacy?

If not, I would label this option as `Use 802.11s for meshing instead of ad-hoc`.